frontend/src/components/eam-configurate-group/helpers/get-columns.helper.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  UsersTableHeader,
+  UsersTableAccessor,
+  StorageKey,
+} from 'common/enums/enums';
+import { Checkbox } from 'components/common/common';
+import { useState } from 'hooks/hooks';
+import { storage } from 'services/services';
+import { Column } from 'react-table';
+
+const selected_workers = new Set();
+
+const checkboxHandler = ({ row }: unknown): JSX.Element => {
+  const [isChecked, setIsChecked] = useState(false);
+  if (isChecked) {
+    selected_workers.add(row.original.id);
+    storage.setItem(
+      StorageKey.SELECTED_WORKERS,
+      JSON.stringify([...selected_workers.values()]),
+    );
+  } else {
+    selected_workers.delete(row.original.id);
+    storage.setItem(
+      StorageKey.SELECTED_WORKERS,
+      JSON.stringify([...selected_workers.values()]),
+    );
+  }
+  const checkboxHandler = (): void => {
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <div>
+      <Checkbox
+        label={''}
+        isChecked={selected_workers.has(row.original.id)}
+        onChange={checkboxHandler}
+      />
+    </div>
+  );
+};
+
+const getColumns = (): Column[] => {
+  return [
+    {
+      Header: '',
+      accessor: 'check',
+      Cell: checkboxHandler,
```

Comment:
Cell is a component. give a better name for it and move it to the 
`frontend/src/components/eam-configurate-group/helpers/get-columns/cells/*cell-name*` - give a better name for it